sprint-planning/readme.md

```diff
@@ -1,29 +1,83 @@
+- Each story should be able to be completed within the duration of the sprint. Otherwise, the story should be broken up into multiple stories.
+- Each task duration should be somewhere between 2 to 8 hours.
 
 ## Participation
 
-What members of the team should participate?
+**Everyone** in the team should participate in the sprint planning.
+
+**Specific roles:**
+
+- **[Scrum Master](https://www.agilealliance.org/glossary/scrum-master)**:
+  - Facilitates the conversation.
+  - Ensures everyone is heard.
+  - Reminds scrums principles and sprint planning goals if necessary.
+- **[Product owner](https://www.agilealliance.org/glossary/product-owner/)**:
+  - Prior to the sprint planning: performs some [backlog grooming](../backlog-management/grooming/readme.md) to ensure that each story that they want to propose for the new sprint (*) :
+  
+    - Is in the correct position in the backlog, by right priority order.
+    - Contains a clear description.
+    - Contains a clear [definition of done](./../team-agreements/definition-of-done/readme.md).
+    - Does NOT pre assign stories to the future sprint. This is the purpose of the sprint planning.
+  - During the meeting:
+  
+    - Describes to the team the stories that they propose for the sprint.
+
+- **All team members**:
+
+  - Listen to the product owner story description.
+  - Ask questions to make sure everyone understands each story properly.
+  - [Estimate](./estimation/readme.md) the effort for each backlog item, as a team.
+  - Split each story into tasks.
+  - (Optional) self assign first task to team members.
+
+*(\*) some teams find useful to define a **[Definition of ready](./../team-agreements/definition-of-ready/readme.md)** that describes the list of things that needs to be done in each story before the **product owner** can propose it for a **sprint**. The list proposed here is the classic minimal definition of ready.*
 
 ## Impact
 
-What positive impact has been observed as a result of this practice?
+**Sprint planning key benefits:**
+
+- Everyone participates, the entire team is aware of the scope of the sprint.
+- Each team member takes responsibility in the sprint scope by participating in the stories discovery, sign up, prioritization and estimation.
+- Creates a platform to communicate, discover and discuss dependencies.
 
 ## Measures
 
-How might one measure the impact of this ceremony positive or negative?
+- How many stories needed more work before being presented (and were rejected because they were not clear enough)?
+- How realistic was the estimation of effort?
+- Team satisfaction (can be assessed during the retrospective): does everyone in the team feel included in sprint planning?
 
 ## Facilitation Guidance
 
-How might one go about running the ceremony? Is there a general pattern that the ceremony should follow?
+**Prior to the meeting:**
+
+- [ ] Set sprint goal.

```

Comment:
These check boxes feel a bit misplaced here as they are not used in the rest of the document.  Without them it would follow the same form as the rest of the text

Reply:
Agreed, removed in upcoming change